### ISSUE TITLE
Microfocus UFT 

### ISSUE THREAD

**Issue by Harry-repo** [MicroFocus/ADM-TFS-Extension#47]
![Screenshot (30)](https://user-images.githubusercontent.com/81159862/142822852-7c4230eb-aed2-4bf9-9969-b50a3a989018.png)

Hi All,  I am trying to integrate the azure devops and UFT one I have followed all the procedure Downloading UFT.zip , Agent ,confirgation in interactive mode , ran unpack.ps1 and Downloaded the UFT one extension. But iam facing an error when i use the 1) Run from the file system task  2021-11-22T07:42:35.0796748Z ##[section]Starting: Run From File System 2021-11-22T07:42:35.1640279Z ============================================================================== 2021-11-22T07:42:35.1640878Z Task : UFT One File System Run 2021-11-22T07:42:35.1641295Z Description : Run UFT One tests from file system 2021-11-22T07:42:35.1641484Z Version : 2.0.3 2021-11-22T07:42:35.1641796Z Author : Micro Focus 2021-11-22T07:42:35.1642276Z Help : https://admhelp.microfocus.com/uft/en/latest/UFT_Help/Content/UFT_Tools/Azure_DevOps_Extension/uft-azure-devops-run-local.htm 2021-11-22T07:42:35.1642678Z ============================================================================== 2021-11-22T07:42:36.5517661Z ##[error]Cannot bind argument to parameter 'Path' because it is null. 2021-11-22T07:42:36.6889830Z ##[section]Finishing: Run From File System  Kindly guide me on this where am i going wrong.  thanks and regards Harish
--

**Comment by PaulaPaltinisanu** [comment]
Hello Harish,

Thank you for reaching out to us!

The error message you are receiving is related to certain files (within UFTWorking folder) not being fully accessible.

Could you please start by checking if the UFT_LAUNCHER environment variable has been added into your agent's machine, as well as if it's pointing into the UFTWorking folder?

If this step has already been completed successfully, please check if any of the executables (especially UFTWorking/bin/PSModule.dll) are unblocked. You can check this by accessing the file's property. If they are indeed blocked, please proceed to unblock them and rerun your pipeline.

Looking forward to hearing from you.
Thank you!

**Comment by PaulaPaltinisanu** [comment]
Hello,

Due to the lack of activity, this issue will closed.
If you happen to encounter any problems, please raise a new issue.

Thank you!
